fabfile.py

```diff
@@ -11,7 +13,7 @@
 env.hosts = ['core@ec2-52-58-89-242.eu-central-1.compute.amazonaws.com']
 env.disable_known_hosts = True
 env.colorize_errors = True
-env.command_timeout = 60 * 5
+env.command_timeout = 60 * 10
```

Comment:
This controls how long fabric will wait before it timesout, so integration tests can only take as long as this (in seconds) - the new integration test ups the run time to ~6 minutes, so I increased it, as well as adding a little bit of breathing room.